pkg/peer/messages.go

```diff
@@ -38,3 +39,15 @@ type DataChannelMessage struct {
 }
 
 type DataChannelAvailable struct{}
+
+type ForwardRTCP struct {
```

Comment:
I would probably name it as `RTCPReceived` or something. After all, these are messages, not commands that the `Peer` sends, i.e. all other commands read like "Hey, I've got a new ICE candidate { candidate }` or "Hey, I've published a new track { track }", so I think it would make sense to call this message also in a similar way to keep consistency (and then it's up to the receiver to decide what to do with them, e.g. forward, or maybe process them in a different way depending on the application logic).